docker/gemini/image

```diff
@@ -1 +1 @@
-scylladb/hydra-loaders:gemini-1.8.0
+scylladb/hydra-loaders:gemini-v1.8.0
```

Comment:
It has typo, must be `gemini-v1.8.2`

Reply:
this file should be remove, nothing uses it.

Reply:
I changed it by mistake, but it has no meaning anyway.